longevity_tombstone_gc_test.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+import time
+from functools import partial
+
+from longevity_twcs_test import TWCSLongevityTest
+from sdcm.utils.common import ParallelObject
+from sdcm.utils.sstable_utils import SstableUtils
+
+
+class TombstoneGcLongevityTest(TWCSLongevityTest):
+    keyspace = 'scylla_bench'
+    table = 'test'
+    ks_cf = f'{keyspace}.{table}'
+    propagation_delay = 60 * 4  # setting a value shorter than the default, of 4 minutes
```

Comment:
comment and value don't fit... it seems to be exactly 4 minutes, but the comment says we are setting a value shorter than 4 minutes (the default) IIUC

Reply:
fixed the comment.